aliases/available/general.aliases.bash

```diff
@@ -72,7 +72,7 @@ alias rd='rmdir'
 alias xt='extract'
 
 # sudo editors
-alias svim='sudo ${VISUAL:-vim}'
+alias svim='sudoedit'
 alias snano='sudo nano'
```

Comment:
First let me start by saying Thanks for taking the time to make a PR !

I see value in using `sudoedit`  trying to hep keep the user on rails, but if a user is in an environment were they are not allowed to run `sudo vim`, then they can quite easily override this alias in their startup profile.

/start rant

Although I'm not happy that, in both of these versions, the alias `svim` is **not** guaranteed to try to actually open `vim`, the old version at least does follow a precedent set earlier in the file (ie.  `${VISUAL:-vim}`)

BUT if the alias is going to be called `svim` then, IMO, it has a responsibility to invoke VIM.

Turning `svim` into a simple alias for `sudoedit` doesn't feel very useful.

Also, ignoring the `snano` alias on the *very next line* feels like you're not taking a full-picture look at the problem you want to solve.

/stop rant

If I were to take an honest look at using this, I would probably try something like the following:
```
alias sedit='sudoedit'
alias svim='SUDO_EDITOR=vim sudoedit'
alias snano='SUDO_EDITOR=nano sudoedit'
```

*NOTE*: `sudoedit` can be configured to:
* Let the user fully specify the editor via environment variables
* Limit which editors are honored in environment variables
* Specify the exact editor to use, regardless of user's wishes

So even here, `SUDO_EDITOR` is just a suggestion/wish that may not be honored.

/start 2nd rant

Additionally, this file also already contains an `edit` alias which completely breaks the pattern:
```
alias edit='${EDITOR:-${ALTERNATE_EDITOR:-nano}}'
```
Ignores `VISUAL`, and defaults to nano instead of vim

At the very least this should probably be:
```
alias edit=${VISUAL:-${EDITOR:-${ALTERNATE_EDITOR:-nano}}}
```
/stop 2nd rant


TL;DR:
* I don't know if there's enough value to warrant a change
* If its an alias for a named editor, it should invoke _that_ editor
* We might want to re-evaluate our view on editor resolution project-wide

Reply:
@davidpfarrell after reading your argumentation (you "rant" is ok for me 😉 ), I agree that there's enough value to warrant a change. But you and anybody using `sudo vim` should know that there's big security issue.

❓ So do you agree and let me change this PR to an updated comment within the file?

Regarding `nano` - I never liked and used `nano` and I don't know if there's also this security issue. Therefore I didn't focus on it.

Reply:
Without diving into the discussion about `sudoedit`, I'd rather see these aliases removed than modified, given the larger context.

Reply:
@davidpfarrell , I agree with @cornfeedhobo that we should remove these `sudo` commands at all. They are not really neccessary or "helpful" at all… Do you agree so I can change my PR?

Reply:
Fixed in https://github.com/Bash-it/bash-it/commit/129340d24d8c75d1eb3e801487da35d56bb75ac2